Classes/Controller/AbstractController.php

```diff
@@ -124,6 +125,12 @@ protected function initialize(RequestInterface $request): void
         $this->pageUid = (int) GeneralUtility::_GET('id');
         $this->requestData['page'] = $this->requestData['page'] ?? 1;
 
+        if (GeneralUtility::makeInstance(Typo3Version::class)->getMajorVersion() >= 12) {
+            if ($request->getAttribute('applicationType') === 1) {
```

Comment:
@beatrycze-volk, @sebastian-meyer, it looks like this code is not in the main branch. Is it not needed there?